environment.default

```diff
@@ -0,0 +1,6 @@
+# Default environment variables for cellxgene development
+set -a
+CXG_SERVER_PORT=5000
```

Comment:
this used to be `5005`.  Is there a good reason to change the default port number?  I'm just worried about unintended downstream impact on other dependent repos.

Reply:
Actually, further testing reveals that this is a bug.  It prevents the front-end hot loading server from working.   (ie, run `make start-frontend`, and the JS code won't find the cellxgene backend).  Definitely needs to be changed to 5005.